cmd/cli/app.go

```diff
@@ -41,6 +41,7 @@ func newApp() *cli.App {
 	app.Usage = "command line utility"
 	app.EnableBashCompletion = true
 	app.Flags = common.GlobalFlags
+	app.Before = common.SetupGlobalLogger
```

Comment:
Does this work? `app.Before` accepts `(c *cli.Context, printLvl bool)` functions?

Reply:
I was just wondering because `app.Before` requires a type `BeforeFunc` and `common.SetupGlobalLogger` does not match the function type https://pkg.go.dev/github.com/urfave/cli#BeforeFunc

Reply:
its v2: https://pkg.go.dev/github.com/urfave/cli/v2#BeforeFunc

Reply:
Ah of course its v2 sorry for the wrong link. Still stragen :sweat_smile: